front_end/html/exercise_submissions.html

```diff
@@ -9,15 +9,23 @@ <h4 style="display: inline-block;">Submissions for {{ exercise_basics["title"] }
     </div>
 
     {% for student in exercise_submissions %}
-        {% if student[1]["passed"] %}
-            <button type="button" class="clickable collapsible">{{ student[1]["name"] }}</button>
+        {% if student[1]["score"] > 60 %}
```

Comment:
I used 'score > 60' rather than 'passed' because if a user passes an exercise and then submits again incorrectly they will get a message that says "student has not passed yet" or something.

Reply:
Thanks for addressing this. However, a score of 60 will not always equate to passing. A better solution would be to make sure that the passed field will be true if they have **ever** passed the exercise, rather than whether their most recent submission was passing.

Will you please dig a little deeper into this to see how the passing values are determined. It may be a quick tweak, or it may require a more advanced change to the SQL. If it's the latter, I can help with that.

Reply:
I didn't know of a way to fetch only the most recent record while scanning through past records for a passing submission using only SQL, (although if there is such a way I'd love to learn!) so I changed the SQL to fetch all submissions instead of grouping them by student and then parsed through them using python to achieve the desired result.

I looked over the pull request again after these changes and it should be ready for review whenever you're ready!